vcstools/gfit.py

```diff
@@ -24,24 +26,32 @@ class gfit:
         The pulse profile to be fit
     max_N: int
         The maximum number of gaussians to attempt to fit
-        Default - 6
+        Default - 10
     plot_name: str
         The name of the output plot. Can be set with gfit.plot_name. If unsupplied, will use a generic name
         Default - None
-    clip_type: str:
-        The verbosity of clipping. Choose between 'regular', 'noisy' or 'verbose'.
-        Default: 'regular'
+    conponent_plot_name: str
+        The name of the plot for the component plot that illustrates how profile components are chosen. If 
+        unsupplied, will not plot. Only applicable if on_pulse_range unsupplied.
+        Default - None
+    scattering_threshold: float
+        The threshold for which any tau value greater will be deemed scattered. 
+        Default - 40
```

Comment:
This needs units, is it percent or ms?

Reply:
I have no idea. It's just a number that looks like it works. Which is fine if it's not modelling anything physical